docs/wxm-devices/D1-bundle/D1-connect-and-claim.mdx

```diff
@@ -0,0 +1,202 @@
+---
+id: D1-connect-and-claim
+title: Connect and Claim Station
+sidebar_label: Connect and Claim Station
+slug: /wxm-devices/D1-bundle/D1-connect-and-claim
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+## WG1200 Gateway
+
+<img object-fit="contain" src={useBaseUrl("/img/home/wg1200-gateway.png")} />
+
+The weather station comes together with the WG1200 Gateway.
+The WG1200 Gateway receives all the meteorological data plus the location of the weather station, which are measured and transmitted every 16 seconds by the outdoor sensor
+and displays them on its screen.
+
+
+In this chapter, you will find guidance to set up the WG1200 Gateway.
+<hr/>
+
+## Prerequisites
+
+1. The station must be fully assembled (parts and batteries).
+1. A working smartphone with connection to the internet.
+3. A reliable 2.4 GHz WiFi network (**5 GHz is not supported**) for the WG1200 Gateway to connect to.
+
+<hr/>
+
+
+
+## Step by Step Instructions
+
+
+
+### Connect WG1200 Gateway to power source and turn the gateway on
+
+
+  
+- The WG1200 Gateway must be connected to a power source at all times.
+
+
+:::caution
+If WG1200 Gateway turns off, you will not receive $WXM rewards, as you will not be able to share data with the WeatherXM network. **This will result in no $WXM rewards!**
+WG1200 does not have a battery, so it must be connected to a power source at all times.
+:::
+
+
+<hr/>
+
+
+
+
+### Change Frequency (Optional)
+
+
+The D1 Gateway's frequency must be set to match the frequency of your weather station so it can listen for the weather data it transmits. There are 2 frequencies available:
+- Model A: 868 MHz (default)
+- Model B: 915 MHz
+
+:::note
+**If your weather station is 868 MHz, you don't need to change anything as it is configured to this value by default.**
+If unsure, check the sticker under the station or the box that came in it.
+:::
+
+
+<hr/>
```

Comment:
Why are there so many empty lines in these lines? Remove the duplicate ones and leave just one empty line between different tags.